src/main/resources/config/application-prod.yml

```diff
@@ -131,3 +131,16 @@ jhipster:
             queue-size: 512
     audit-events:
         retention-period: 120 # Number of days before audit events are deleted.
+
+# Properties to be exposed on the /info management endpoint
```

Comment:
please add a Github documentation page where you explain the guided tutorials and how to set them up

Reply:
@krusche I have created a Github issue for that and will add the documentation in a follow-up PR https://github.com/ls1intum/Artemis/issues/1055